Makefile

```diff
@@ -77,7 +77,8 @@ toolset: $(TOOLSET_PATH)
 $(TOOLSET_PATH): $(BINARIES_PATH)/dotnet-cli
 	pushd $(TOOLSET_SRC_PATH) ; \
 	$(BINARIES_PATH)/dotnet-cli/dotnet restore && \
-	$(BINARIES_PATH)/dotnet-cli/dotnet publish -o $(TOOLSET_PATH)
+	$(BINARIES_PATH)/dotnet-cli/dotnet publish -o $(TOOLSET_PATH) && \
+	sed -i -e 's/Microsoft.CSharp.Targets/Microsoft.CSharp.targets/g' $(TOOLSET_PATH)/Extensions/Microsoft/Portable/v5.0/Microsoft.Portable.CSharp.targets
```

Comment:
@jaredpar Tracking bug was here: https://github.com/dotnet/roslyn/issues/9641

But the NuGet package we created to hold the portable targets hasn't been updated. I'm not sure if that update is even planned, given the new track for portable projects.